README.rst

```diff
@@ -3,4 +3,30 @@ The conservation element and solution element (CESE) method
 ===========================================================
 
 .. image:: https://mybinder.org/badge_logo.svg
-  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
\ No newline at end of file
+  :target: https://mybinder.org/v2/gh/yungyuc/turgon/master?filepath=notebook
+
+Build Turgon on OS X
+====================
+Clone from https://github.com/yungyuc/turgon.git:
+ `$ git clone https://github.com/yungyuc/turgon.git`
+
+Turgon needs following packages:
```

Comment:
You are making a 'definition list' in this section.  It doesn't look like a good way to format the sequence of actions.  Could you change to use simple paragraphs?

Note that there should be a blank line between `.. code-block::` and it's previous line.